debian/control

```diff
@@ -40,27 +49,39 @@ Depends: ${misc:Depends},
          python3-pkg-resources,
          ${extra:Depends}
 Recommends: ubuntu-pro-client-l10n
-Description: management tools for Ubuntu Pro
- Ubuntu Pro is the professional package of tooling, technology
- and expertise from Canonical, helping organisations around the world
- manage their Ubuntu deployments.
- .
- Subscribers to Ubuntu Pro will find helpful tools for accessing
- services in this package.
+Breaks: ubuntu-advantage-tools (<<31~)
+Replaces: ubuntu-advantage-tools (<<31~)
+Provides: ubuntu-advantage-tools
+Description: Management tools for Ubuntu Pro
+ Ubuntu Pro is a suite of additional services provided by Canonical on
+ top of Ubuntu. Whether you’re an enterprise customer deploying systems
+ at scale or want security patching for your personal Ubuntu LTS
+ at home, the Ubuntu Pro Client (pro) is the command-line tool that
+ will help you manage the services you need.
 
 Package: ubuntu-pro-client-l10n
 Architecture: any
-Depends: ${misc:Depends}, ubuntu-advantage-tools (>=30~)
+Depends: ${misc:Depends}, ubuntu-pro-client (>=31~)
```

Comment:
We should be able to automate these versioned dependencies by using `${binary:Version}`, see the deb-substvars(5) manpage.